iStore/Login/RegisterButton.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  RegisterButton.swift
+//  iStore
+//
+//  Created by Mohamed Sayed on 13.03.24.
+//
+
+import SwiftUI
+
+struct RegisterButton: View {
+    
+    @EnvironmentObject var navigationCoordinator: NavigationCoordinator
+    
+    var body: some View {
+        Button("Register") {
+            navigationCoordinator.switchView = .registerView
```

Comment:
This means that the button always will do this line. Why did you make a separate view the button?

Reply:
I wanted it to always navigate to the registerView but the backend doesn't support user registration so, I replaced it with button ("Continue as aGuest instead")